README.md

```diff
@@ -35,6 +35,7 @@ VMware community collection depends upon following third party libraries:
 
 * [`Pyvmomi`](https://github.com/vmware/pyvmomi) >= 6.7.1.2018.12
 * [`vSphere Automation SDK for Python`](https://github.com/vmware/vsphere-automation-sdk-python/)
+* [`vSAN Management SDK for Python`](https://code.vmware.com/web/sdk/7.0%20U2/vsan-python)
```

Comment:
```suggestion
* [`vSAN Management SDK for Python`](https://code.vmware.com/web/sdk/vsan-python)
```

I think it might be better to link to the generic vSAN Management SDK for Python, not to the specific 7.0U2 version. We don't link to a specific PyVmomi or vSphere Automation SDK version, either.

Reply:
Indeed, I didn't know that can remove the version from the URL.
Thank you @mariolenz for reviewing!